gooey/cli.py

```diff
@@ -0,0 +1,35 @@
+'''The command line interface.'''
+
+import sys
+from importlib import import_module
+from importlib.metadata import entry_points
+
+from gooey import Gooey
+
+
+def main(args=None):
+    args = args or sys.argv[1:]
+    if not args:
+        # without args we assume we want to run gooey on gooey itself
+        # TODO: rewrite using argparse
```

Comment:
Running Gooey on itself requires rewriting this function with an argparse parser.

Reply:
I'll leave it up to you ^_^ 

I'd be fine going sans additional configuration for the initial release of this -- let it "bake" a bit and see if people need the functionality. On the flip side, if you want to throw a parser in there so people can control things that's fine, too. You may be able to even generate a options for all of the [params](https://github.com/chriskiehl/Gooey/blob/master/gooey/python_bindings/parameters.py) directly from the type definition by looking through its `__annotations__` (in theory).